src/api/pipelineTaskHelpers.ts

```diff
@@ -141,7 +141,7 @@ export const getAllPipelineTasks = (forms: ImportWizardFormState, namespace: str
     const { targetPvcName, storageClass, capacity, verifyCopy } = editValues; // TODO where to put verifyCopy?
     console.log('TODO: use verifyCopy flag!', pvc.metadata?.name, verifyCopy);
     return {
-      name: 'transfer-pvc',
+      name: 'transfer-pvc-' + pvc.metadata?.name,
```

Comment:
As a rule I try to avoid using `+` for string concatenation because it can have unexpected effects when working with non-string types. That doesn't apply here but I try to stay consistent with it in the codebase.

Can you replace this with a string template literal?
```suggestion
      name: `transfer-pvc-${pvc.metadata?.name}`,
```